minimal-option

```diff
@@ -0,0 +1,120 @@
+gnome-characters gnome-control-center
+gnome-control-center-data
+gnome-control-center-faces
+gnome-desktop3-data
+gnome-font-viewer
+gnome-getting-started-docs
+gnome-initial-setup
+gnome-keyring gnome-keyring-pkcs11
+gnome-logs
+gnome-mahjongg
+gnome-menus
+gnome-mines
+gnome-online-accounts
+gnome-power-manager
+gnome-screenshot
+gnome-session-bin
+gnome-session-canberra
+gnome-session-common
+gnome-settings-daemon
+gnome-settings-daemon-common
+gnome-shell
+gnome-shell-common
+gnome-shell-extension-appindicator
+gnome-shell-extension-desktop-icons
+gnome-shell-extension-ubuntu-dock
+gnome-startup-applications
+gnome-sudoku
+gnome-system-monitor
+gnome-terminal
+gnome-terminal-data
+gnome-themes-extra
+gnome-themes-extra-data
+gnome-todo
+gnome-todo-common
+gnome-user-docs
+gnome-video-effects
+gdm3
+nautilus
+nautilus-data
+nautilus-extension-gnome-terminal
+nautilus-sendto
+nautilus-share
+pinentry-gnome3
+yaru-theme-gnome-shell
+yaru-theme-gtk
+yaru-theme-icon
+yaru-theme-sound
+zenity
+zenity-common
+snapd
+software-properties-common
+software-properties-gtk
+sound-theme-freedesktop
+yelp
+yelp-xsl
+x11-apps
+totem
+totem-common
+totem-plugins
+xwayland
+x11-common
+x11-session-utils
+x11-utils
+x11-xkb-utils
+x11-xserver-utils
+xauth
+xbitmaps
+xbrlapi
+xcursor-themes
+xdg-desktop-portal-gtk
+xorg
+xorg-docs-core
+xserver-common
+xserver-xephyr
+xserver-xorg
+xserver-xorg-core
+xserver-xorg-input-all
+xserver-xorg-input-libinput
+xserver-xorg-input-wacom
+xserver-xorg-legacy
+xserver-xorg-video-all
+xserver-xorg-video-amdgpu
+xserver-xorg-video-ati
+xserver-xorg-video-fbdev
+xserver-xorg-video-intel
+xserver-xorg-video-nouveau
+xserver-xorg-video-qxl
+xserver-xorg-video-radeon
+xserver-xorg-video-vesa
+xserver-xorg-video-vmware
+whoopsie
+whoopsie-preferences
+ubuntu-wallpapers
+ubuntu-report
+ubuntu-release-upgrader-gtk
+shotwell-common
+shotwell
+rhythmbox
+rhythmbox-data
+rhythmbox-plugin-alternative-toolbar
+rhythmbox-plugins
+remmina
+remmina-common
+remmina-plugin-rdp
+remmina-plugin-secret
+remmina-plugin-vnc
+poppler-data
+poppler-utils
+apport-gtk
+apport
+apport-symptoms
+baobab
+dbus-x11
```

Comment:
dbus x11 should remain installed, unless it is installed when installing Xorg?

Reply:
This is not needed as dbus-user-session is all xorg needs plus it gets installed if needed later: https://packages.ubuntu.com/source/kinetic/dbus

Reply:
But yeah this could be removed.